tekton/templates/test-teardown-task.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: test-teardown
+  namespace: tektoncd
+  annotations:
+    sidecar.istio.io/inject: "false"
+spec:
+  workspaces:
+  - name: junit-artifacts
+  inputs:
+    params:
+    - name: workflow-name
+      type: string
+      description:
+        Path to the directory with artifacts to upload to GCS.
+    # params below are from prow jobs.
+    - name: test-target-name
+      type: string
+      description:
+        Test target name from Prow ENV. Should be provided at runtime.
+    - name: repo-owner
+      type: string
+      description:
+        Testing repo owner from Prow ENV.
+    - name: prow-job-id
+      type: string
+      description:
+        Prow job ID from Prow ENV.
+    - name: job-type
+      type: string
+      description:
+        Job type from Prow ENV.
+    - name: job-name
+      type: string
+      description:
+        Job name from Prow ENV.
+    - name: repo-name
+      type: string
+      description:
+        Testing repo name from Prow ENV.
+    - name: pull-number
+      type: string
+      description:
+        Pull number from Prow ENV.
+    - name: build-id
+      type: string
+      description:
+        BUILD_ID from Prow ENV.
+    resources:
+    - name: kf-testing-repo
+      type: git
+      targetPath: src/kubeflow/testing
+  steps:
+  - name: copy-artifacts
```

Comment:
Can we use Tekton resources to upload files to GCS as opposed to having a separate task for that?

Reply:
just confirmed we could.